internal/pkg/service/stream/storage/test/file.go

```diff
@@ -80,9 +80,13 @@ func NewFileOpenedAt(openedAtStr string) model.File {
 			UploadCredentials:           &keboola.FileUploadCredentials{},
 			UploadCredentialsExpiration: utctime.From(openedAt.Time().Add(time.Hour)),
 		},
-		TargetStorage: target.File{
-			TableID:    keboola.MustParseTableID("in.bucket.table"),
-			StorageJob: nil,
+		TargetStorage: target.Target{
+			Table: target.Table{
+				Keboola: target.KeboolaTable{
+					TableID:    keboola.MustParseTableID("in.bucket.table"),
+					StorageJob: nil,
+				},
+			},
 		},
```

Comment:
Improved forward compatibility.

Now, we are importing data to the Keboola table, using a Storage Job.
There are some limits, it is safe to run at most 1 Storage Job each `30s`.

In the future, we will probably write directly to the table. So there will be several configuration options.